src/outspin/unix.py

```diff
@@ -53,6 +53,8 @@
     "\x1b[1;10B": "shift+alt+down",
     "\x1b[1;10C": "shift+alt+right",
     "\x1b[1;10D": "shift+alt+left",
+    "\x1b[1;5C": "ctrl+left",
+    "\x1b[1;5D": "ctrl+right",
```

Comment:
Are these in the right order? Judging based on the other ones it seems like "right" should get `C` and "left" should get `D`

Reply:
oh wait true

Reply:
why does the test pass though :sob: